openbb_terminal/core/models/fmp.py

```diff
@@ -0,0 +1,840 @@
+    ) -> pd.DataFrame:
+        """Get historical full financial statement as reported
+
+        Parameters
+        ----------
+        symbol : str
+            Symbol to get data for
+        quarter : bool, optional
+            Flag to get quarterly data, by default False
+        limit : int, optional
+            Number of statements to get, by default 200
+
+        Returns
+        -------
+        pd.DataFrame
+            Historical statement dataframe with columns as the reported dates
+
+        Examples
+        --------
+        >>> from openbb_terminal.sdk import openbb
+        >>> statements = openbb.fmp.statements("AAPL")
+        """
+        if quarter:
+            url = f"{cls._BASE_URL()}financial-statement-full-as-reported/{symbol}?period=quarter"
+        else:
+            url = f"{cls._BASE_URL()}financial-statement-full-as-reported/{symbol}?"
+        r = request(url + "&limit=" + str(limit) + "&apikey=" + cls._KEY())
+        return pd.DataFrame(cls.validate_request(r)).T
+
+    @classmethod
+    def key_metrics(
+        cls, symbol: str, quarter: bool = False, limit: int = 200
+    ) -> pd.DataFrame:
+        """Get historical key metrics
+
+        Parameters
+        ----------
+        symbol : str
+            Symbol to get data for
+        quarter : bool, optional
+            Flag to get quarterly data, by default False
+        limit : int, optional
+            Number of statements to get, by default 200
+
+        Returns
+        -------
+        pd.DataFrame
+            Historical key ratios
+
+        Examples
+        --------
+        >>> from openbb_terminal.sdk import openbb
+        >>> key_ratios = openbb.fmp.key("AAPL")
+        """
+        if quarter:
+            url = f"{cls._BASE_URL()}key-metrics/{symbol}?period=quarter&"
+        else:
+            url = f"{cls._BASE_URL()}key-metrics/{symbol}?"
+        r = request(url + f"apikey={cls._KEY()}&limit={limit}")
+        return pd.DataFrame(cls.validate_request(r)).T
+
+    @classmethod
+    def ratios(cls, symbol: str, quarter=True, limit: int = 200) -> pd.DataFrame:
+        """Get historical ratios
+
+        Parameters
+        ----------
+        symbol : str
+            Symbol to get data for
+        quarter : bool, optional
+            Flag to get quarterly data, by default False
+        limit : int, optional
+            Number of statements to get, by default 200
+
+        Returns
+        -------
+        pd.DataFrame
+            Historical ratios
+
+        Examples
+        --------
+        >>> from openbb_terminal.sdk import openbb
+        >>> ratios = openbb.fmp.ratios("AAPL")
+        """
+        url = (
+            f"{cls._BASE_URL()}ratios/{symbol}?period=quarter&"
+            if quarter
+            else f"{cls._BASE_URL()}ratios/{symbol}?"
+        )
+        r = request(url + f"apikey={cls._KEY()}&limit={limit}")
+        return pd.DataFrame(cls.validate_request(r)).T
```

Comment:
This should not be transposed. 
<img width="847" alt="image" src="https://user-images.githubusercontent.com/40023817/222856285-5b4db4f7-5264-4618-b70c-8425644f8f72.png">